modules/custom/az_dam/config/quickstart/node.type.ib_demo_page.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 64a83cac-4f3c-4eb5-aa4e-d0847332b015
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.